server/datastore/mysql/migrations/tables/20250424162727_AddSoftwareCategories.go

```diff
@@ -0,0 +1,55 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20250424162727, Down_20250424162727)
+}
+
+func Up_20250424162727(tx *sql.Tx) error {
+	_, err := tx.Exec(`CREATE TABLE IF NOT EXISTS software_categories  (
+    		id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    		name VARCHAR(63) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+			UNIQUE KEY idx_software_categories_name (name)
+			)`)
+	if err != nil {
+		return fmt.Errorf("failed to create software_categories table: %w", err)
+	}
+
+	_, err = tx.Exec(`CREATE TABLE IF NOT EXISTS software_installer_software_categories  (
+    		id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    		software_category_id INT UNSIGNED NOT NULL,
+			software_installer_id INT UNSIGNED NOT NULL,
+			created_at DATETIME(6) NULL DEFAULT NOW(6),
+			FOREIGN KEY (software_installer_id) REFERENCES software_installers (id) ON DELETE CASCADE
```

Comment:
Need a FK to category here too. I know we don't need this _yet_ but makes life easier to go ahead and add it for when we need to munge the category list.